Feature/webhooks-applications (#2336)
mercadopago/devsite-docs: guides/additional-content/homologator/homologator.en.md

## How to homologate my integration?

To measure the quality of your application, follow the steps below.

----[mla, mlm, mlu, mco, mlc, mpe]----

> WARNING
>
> Importante
>
> Before starting the evaluation, make sure that the application has been approved in the production environment, including the completion of at least one productive payment. <br><br>
> <br><br>

> > It is necessary that it be an application where there is a product to be integrated from those where the measuring tool is available. For now, the integration quality measuring tool is only available for integrations with [Checkout Pro,](/developers/en/docs/checkout-pro/landing) [Checkout API](/developers/en/docs/checkout-api/landing), [Checkout Bricks](/developers/en/docs/checkout-bricks/landing) and [Mercado Pago Point.](/developers/en/docs/mp-point/landing)

Reviewer: ```suggestion
> It is necessary that the application is registered with an integration of a product from those where the measuring tool is available. For now, the integration quality measuring tool is only available for integrations with [Checkout Pro,](/developers/en/docs/checkout-pro/landing) [Checkout API](/developers/en/docs/checkout-api/landing), [Checkout Bricks](/developers/en/docs/checkout-bricks/landing) and [Mercado Pago Point.](/developers/en/docs/mp-point/landing)
```

------------
----[mlb]----

> WARNING
>

> > Importante

Reviewer: ```suggestion
> Important
```

>
> Before starting the evaluation, make sure that the application has been approved in the production environment, including the completion of at least one productive payment.  <br><br>
> <br><br>

> > It is necessary that it be an application where there is a product to be integrated from those where the measuring tool is available. For now, the integration quality measuring tool is only available for integrations with [Checkout Pro,](/developers/en/docs/checkout-pro/landing) [Checkout Transparente](/developers/en/docs/checkout-api/landing), [Checkout Bricks](/developers/en/docs/checkout-bricks/landing) and [Mercado Pago Point.](/developers/en/docs/mp-point/landing)

Reviewer: ```suggestion
> It is necessary that the application is registered with an integration of a product from those where the measuring tool is available. For now, the integration quality measuring tool is only available for integrations with [Checkout Pro,](/developers/en/docs/checkout-pro/landing) [Checkout Transparente](/developers/en/docs/checkout-api/landing), [Checkout Bricks](/developers/en/docs/checkout-bricks/landing) and [Mercado Pago Point.](/developers/en/docs/mp-point/landing)
```
